dnaStreaming/services/credentials_service.py

```diff
@@ -5,7 +5,7 @@
 
 
 def fetch_credentials(config):
-    response = _get_requests().get(config.get_uri_context() + '/accounts/streaming-credentials',
+    response = _get_requests().get(config.get_uri_context() + '/sns-accounts/streaming-credentials',
```

Comment:
maybe I haven't been paying close attention, but we implemented this already? It wasn't part od the other alphaless PR for accounts

Reply:
that's because this is a gateway-level modification